projects/scripts/project-intel.mk

```diff
@@ -1,5 +1,6 @@
 ####################################################################################
-## Copyright 2018(c) Analog Devices, Inc.
+## Copyright (c) 2020 - 2021 Analog Devices, Inc.
```

Comment:
Not sure why this has 2020-2021 when all others have 2018-2021

Reply:
typo - I can fix it.